lib/src/request_processor.cc

```diff
@@ -819,4 +847,37 @@ void RequestProcessor::logLoadedModules() const
     }
 }
 
+//
+// Spool directory purge task (private interface)
+//
+
+void RequestProcessor::spoolDirPurgeTask()
+{
+    auto num_minutes = Timestamp::getMinutes(spool_dir_purge_ttl_);
+    num_minutes *= 1.2;
+    LOG_DEBUG("Starting the task for purging the spool directory every %1% "
+              "minute%2%; thread id %3%",
+              num_minutes, lth_util::plural(num_minutes),
```

Comment:
Uses of `plural` are something we'll want to look for when doing i18n work. We'll want to use Boost.Locale facilities to denote pluralization instead. Nothing to do yet though.